modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/AddObjectField.tsx

```diff
@@ -63,10 +63,20 @@ function ModalAddObjectField({
 		indexedLanguageId: null,
 		listTypeDefinitionExternalReferenceCode: '',
 		listTypeDefinitionId: 0,
+		readOnly: 'false',
+		readOnlyConditionExpression: '',
 		required: false,
 	};
 
 	const onSubmit = async (field: Partial<ObjectField>) => {
+		if (
+			field.businessType === 'Formula' ||
```

Comment:
@danvb112 Similar to the comment above. Could you reorder it like
`field.businessType === 'Aggregation' || field.businessType === 'Formula'`? Thanks :)